app/public/js/ttPerson.js

```diff
@@ -203,7 +209,8 @@ class TTCharacter {
 
 
 var humanImage = requireImage('/gfx/person/personSprite.png');
-var characterMysteryImage = requireImage('/gfx/person/characterMystery.png');
+var characterMysteryImage = requireImage('/gfx/person/characterMystery.png'),
+characterAlienImage = requireImage('/gfx/person/characterAlien.png');
```

Comment:
When you declare variables on multiple lines, please indent subsequent lines like:
```
var a = 1,
    b = 2,
    c = 3;
```
As opposed to:
```
var a = 1,
b = 2,
c = 3;
```

This way it is easier to tell which lines are part of the same declaration.